src/features/Apiexplorer/Schema/SchemaObjectContent.tsx

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import RecursiveProperties from './RecursiveProperties';
+import SchemaDescription from './SchemaDescription';
+import styles from './Schema.module.scss';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+// Header component
+type TSchemaBodyHeader = {
+  key_value: string;
+  type: string;
+  defaultValue: string;
+  pattern: string;
+  title: string;
+  is_open_object: boolean;
+  setIsOpenObject: (boolean) => void;
+  examples: string[];
+  enum;
+};
+type TSchemaObjectContent = {
+  key_value: string;
+  properties: any;
+};
+const SchemaBodyHeader = ({
+  key_value,
+  type,
+  defaultValue,
+  pattern,
+  examples,
+  enum: _enum,
+  title,
+  is_open_object,
+  setIsOpenObject,
+}: TSchemaBodyHeader) => {
+  let typeClassName;
+  switch (type) {
+    case 'number':
+      typeClassName = styles.number;
+      break;
+    case 'array':
+      typeClassName = styles.array;
+      break;
+    case 'integer':
+      typeClassName = styles.integer;
+      break;
+    default:
+      typeClassName = styles.string;
+      break;
+  }
+  if (Array.isArray(type)) {
+    type.map((item) => {
+      if (item !== 'null') {
+        switch (item) {
+          case 'number':
+            typeClassName = styles.number;
+            break;
+          case 'array':
+            typeClassName = styles.array;
+            break;
+          case 'integer':
+            typeClassName = styles.integer;
+            break;
+          default:
+            typeClassName = styles.string;
+            break;
+        }
+      }
+    });
+  }
+
+  const is_not_an_object = type && type !== 'object' && typeof type !== 'object';
+
+  const is_type_with_enumlabel = _enum && is_not_an_object;
+  const is_only_type = !_enum && is_not_an_object;
```

Comment:
make this a lil bit cleaner, it's hard to understand what's going on here. thanks.